constants.py

```diff
@@ -446,3 +454,50 @@
 kCameraServoPGain = 0.15
 kCameraServoIGain = 0.0
 kCameraServoDGain = 0.0
+
+# shooter parameters
+kTurretMotorId = 22
+kShootingMotorId = 23
+kHoodMotorId = 21
+
+kSimTurretMotorPort = 9
+kSimShootingMotorPort = 18
+kSimHoodMotorPort = 19
+
+kSimTurretMinimumLimitSwitchPort = 0
+kSimTurretMaximumLimitSwitchPort = 1
+kSimHoodMinimumSwitchPort = 2
+kSimHoodMaximumSwitchPort = 3
+
+kShootingPIDSlot = 0
+kShootingPGain = 0.2
+kShootingIGain = 0
+kShootingDGain = 0
+
+kShootingMappingFunction = lambda x: -100 * x  # TODO: a real tested algorihm
```

Comment:
Looks good! If we can get some good multipliers, we should have a pretty sweet shooter (although it doesn't seem we've done anything for movement yet: this is good for now tho)